README.md

```diff
@@ -1,32 +1,56 @@
-# `@ubiquity/ts-template`
+# `@ubiquibot/command-start-stop`
 
-This template repository includes support for the following:
+This plugin allows a hunter to begin a task as well as gracefully stop a task without incurring a negative impact on the hunter's XP or karma.
 
-- TypeScript
-- Environment Variables
-- Conventional Commits
-- Automatic deployment to Cloudflare Pages
+## Usage
 
-## Testing
+### Start a task
 
-### Cypress
+To start a task, a hunter should use the `/start` command. This will assign them to the issue so long as the following is true:
 
-To test with Cypress Studio UI, run
+- Price labels are set on the issue
+- The issue is not already assigned
```

Comment:
This reminds me that it would be really interesting in the future to support teams. Where I could write on a new task 

```
/start @Keyrxng 
```

And we would both be assigned (and be eligible to split the reward.)

I know a lot of developers work as lone rangers but it could make the experience more collaborative and exciting if we have a massive tasks marketplace and teams hunt projects to co-op on together.